### ISSUE TITLE
Mermaid stopped working in Quarto

### ISSUE THREAD

**Issue by bkostadi** [quarto-dev/quarto-cli#10787]
### Bug description

Mermaid diagrams in Quarto cannot be compiled into a PDF format. There is no problem with HTML. 
When I try to render the Quarto file with any simple mermaid code into a PDF file, I get 

`==> quarto preview mermaid_test.qmd --to pdf --no-watch-inputs --no-browse` 

and the job runs indefinitely so I have to stop it, and no PDF is produced.  This is what I get on the Posit Cloud. When I did the same thing locally on my computer using the latest versions of everything, I got the error message:

```
==> quarto preview mermaid_pdf_test.qmd --to pdf --no-watch-inputs --no-browse

Error polling connection: error shutting down connection: Socket is not connected (os error 57)
ERROR: TypeError: error sending request for url (http://localhost:9222/json/list): error trying to connect: tcp connect error: Connection refused (os error 61)

Stack trace:
    at async mainFetch (ext:deno_fetch/26_fetch.js:266:12)
    at async fetch (ext:deno_fetch/26_fetch.js:490:7)
    at async externalRequest (file:///Applications/quarto/bin/quarto.js:38336:22)
    at async List (file:///Applications/quarto/bin/quarto.js:38363:21)
    at async Chrome._fetchDebuggerURL (file:///Applications/quarto/bin/quarto.js:38552:37)
    at async Chrome._start (file:///Applications/quarto/bin/quarto.js:38509:25)
```

With HTML format it works just fine, both locally and on the Posit Cloud. 

### Steps to reproduce
`````
---
title: "Mermaid in PDF"
format: pdf
---

## Mermaid

```{mermaid}
flowchart TD
A{bla-bla}
```
`````

### Expected behavior

To produce a PDF with the mermaid diagram. 

### Actual behavior

No PDF gets produced. The process hangs indefinitely. 

### Your environment

A new project on the Posit Cloud. 

### Quarto check output
```
/cloud/project$ quarto check
Quarto 1.5.53
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.2.0: OK
      Dart Sass version 1.70.0: OK
      Deno version 1.41.0: OK
      Typst version 0.11.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.5.53
      Path: /opt/quarto/bin

[✓] Checking tools....................OK
      TinyTeX: (external install)
      Chromium: (not installed)

[✓] Checking LaTeX....................OK
      Using: TinyTex
      Path: /home/r157428/.TinyTeX/bin/x86_64-linux
      Version: 2024

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.8.17
      Path: /tmp/RtmpgDWuTV/rstudio/terminal/python3
      Jupyter: (None)

      Jupyter is not available in this Python installation.
      Install with python3 -m pip install jupyter

[✓] Checking R installation...........OK
      Version: 4.4.1
      Path: /opt/R/4.4.1/lib/R
      LibPaths:
        - /cloud/lib/x86_64-pc-linux-gnu-library/4.4
        - /opt/R/4.4.1/lib/R/library
      knitr: 1.48
      rmarkdown: 2.28

[✓] Checking Knitr engine render......OK
```

**Comment by mcanouil** [comment]
Could you properly format your post using code blocks for code and terminal outputs? Thanks.
If your code contains code blocks, you need to enclose it using more backticks, *i.e.*, usually four ` ```` `.
See <https://quarto.org/bug-reports.html#formatting-make-githubs-markdown-work-for-us>.

**Comment by cscheid** [comment]
This has been fixed in Quarto 1.5.57 (unfortunately it was caused by an upstream change to Chrome that we needed to adapt to). I'm going to go ahead and close this. In the future, please [look for similar reports before filing a new bug](https://github.com/quarto-dev/quarto-cli/issues?q=is%3Aissue+is%3Aclosed+chromium+mermaid). Thanks!
